### PR TITLE
Adds Firebase Database URL field in Configure Section.

### LINKED ISSUE #21: Firebase database URL is hardcoded. This should be configurable.
[This line](https://github.com/opendatakit/skunkworks-parrot/blob/master/src/main/java/Controller/LoginFormController.java#L147) sets the database URL to https://odk-notifications.firebaseio.com/ but this should be configurable by project

### PR DESCRIPTION
**Pull request by DreamyPhobic**
Closes #21

### CODE REVIEW

File: src/main/resources/fxml/Settings.fxml [hunk 1]
```diff
@@ -54,7 +54,21 @@
                     <children>
                         <TextField fx:id="webAPIKeyField" prefHeight="31.0" prefWidth="400.0" />
                     </children>
-                </HBox> <Label text="Dynamic Link Domain">
+                </HBox>
+            <Label text="Database URL">
```
Comment (linl33) [inline src/main/resources/fxml/Settings.fxml]: Database URL could mean a number of things in this context. Pick a different label, maybe Firebase Realtime Database URL or Firebase Database URL.

File: src/main/java/Data/Data.java [hunk 2]
```diff
@@ -30,6 +31,7 @@ public Data(){
               setDYNAMIC_LINK_DOMAIN((String) mainObject.getString("dynamicLinkDomain"));
               setANDROID_APP_PACKAGE_NAME((String) mainObject.getString("packageName"));
               setSYNC_CLIENT_URL((String) mainObject.getString("syncClientURL"));
+              setDATABASE_URL((String) mainObject.getString("databaseURL"));
```
Comment (linl33) [inline src/main/java/Data/Data.java]: The cast to `String` is redundant.

File: src/main/java/Data/Data.java [hunk 3]
```diff
@@ -142,4 +146,12 @@ public String toString() {
                 ", SYNC_CLIENT_URL='" + SYNC_CLIENT_URL + '\'' +
                 '}';
     }
+
+    public String getDATABASE_URL() {
```
Comment (linl33) [inline src/main/java/Data/Data.java]: Please follow the convention of this class and move these getter/setter up where the other getters and setters are.

**Comment by DreamyPhobic** [comment]
@linl33 please review new changes.

File: src/main/java/Data/Data.java [hunk 4]
```diff
@@ -132,12 +132,20 @@ public void setSYNC_CLIENT_URL(String SYNC_CLIENT_URL) {
...
+        this.FIREBASE_DATABASE_URL = FIREBASE_DATABASE_URL;
+    }
+
     @Override
     public String toString() {
         return "Data{" +
                 "SUPERVISOR_ID='" + SUPERVISOR_ID + '\'' +
                 ", WEB_API_KEY='" + WEB_API_KEY + '\'' +
-                ", DATABASE_URL='" + DATABASE_URL + '\'' +
+                ", DATABASE_URL='" + FIREBASE_DATABASE_URL + '\'' +
```
Comment (linl33) [inline src/main/java/Data/Data.java]:
```
", DATABASE_URL='"
```
This string here should be changed as well.

**Pull request merged**